data/Cyprus/House_of_Representatives/sources/morph/openpatata-ids.csv

```diff
@@ -91,3 +91,4 @@ stavrou-eleni,1300a45f-e959-4c03-86ea-0e617453d75a
 stefanou-stefanos,5f7ae1a1-3120-46bb-8b12-3db943c74571
 theologou-anna,f66b51ea-891c-4fbf-adf0-6d78a8ad29d5
 theopemptou-charalampos,bf608732-89a6-467a-802a-bd301d2e72eb
+georgiou-k-giorgos,d8a3a81b-678d-46b7-ba6e-e39e68dd2ecc
```

Comment:
Ah, if it's an issue with the Morph data, then removing the 'georgiou-giorgos' line here won't help — all that will happen then is that when it processes that entry it'll give it another new UUID. If you're going to be changing IDs for people semi-regularly, we should probably make the scraper cope with that in some way.